src/store/getters.ts

```diff
@@ -1081,6 +1082,18 @@ export default {
     return state.stateModel.stateFilingRestoration
   },
 
+  getRelationships (state: StateIF): RelationshipTypes[] {
+    return state.stateModel.restoration.relationships
+  },
+
+  getRelationshipValid (state: StateIF): boolean {
+    return state.stateModel.validationFlags.flagsCompanyInfo.isValidRelationship
+  },
+
+  getIsRestorationTypeCourtOrder (state: StateIF): boolean {
+    return state.stateModel.restoration.courtOrder?.fileNumber !== null
```

Comment:
Same comment as above.

Also, is it possible the file number is undefined or empty string? (What is the value if you enter court order and then change the radio button to unset it?

Reply:
This getter method has been moved to `store.ts` as part of the Pinia upgrade.  Like `getRelationships()` (see comment above) I think it's important that our business logic not reside in components and not be duplicated.

Reply:
My second point here is that the comparison is against null only, but if the courtOrder object doesn't exist then the first part of the comparison will be undefined (which is not equal to null).

I'd be tempted to use `!` to check this instead.

Reply:
I like that!  Fixed